packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -405,3 +405,60 @@ instance:
   });
 </script>
 ```
+
+#### Tracking the Add2Cart event from PDP
```

Comment:
```suggestion
### Tracking the Add2Cart event from PDP
```

This is the correct heading for a new section to be in the same level as the other `Notes on X Archetype integration`